src/main/java/ch/uzh/ifi/seal/soprafs20/game/Game.java

```diff
@@ -40,39 +42,173 @@
     @Enumerated
     WeatherState weatherState;
 
-    Game(){
+    Game(User user){
         this.board = new CasualBoard();
         this.deck = new Deck();
         this.gameState = GameState.PENDING;
+        Player hostPlayer = this.transformUserIntoPlayer(user);
+        this.players.add(hostPlayer);
```

Comment:
- `User user` is not very descriptive.
- The `hostPlayer` should also be saved permanently.